pages/dkp/kommander/2.2/release-notes/platform-application-versions/index.md

```diff
@@ -0,0 +1,93 @@
+---
+layout: layout.pug
+navigationTitle: Supported Platform Applications
+title: Supported Platform Applications
+menuWeight: 60
+excerpt: Upgrade Platform applications
+beta: false
+---
+ 
+<!-- markdownlint-disable MD0013 MD030 MD034 -->
+
+Upgrading to the latest version of DKP adapts your Platform Applications to the following versions:
+
+| Platform Application in 2.x | Platform Application in 2.2 |
```

Comment:
> NIT: Should we be more specific than 2.x? I mean, technically 2.x could match 2.2, right?

Is this for upgrading?? I don't know if we need 'what it was before for every version'. Only what it was before on the most previous version.

Reply:
Yep, I think that is a good idea. We would then have all information centralized in the RNs and refer to them anytime we encourage checking anything version-related. In that case, I would close this PR and work with @Colin1841 and eng to have that table put and updated on the RN. Sounds good?

Reply:
So if we have this only in release notes, it would be kept up to date with each patch version, right? That way people could reference this information there? Or do we also need what the default version was/is with a major/minor release? I'm not sure.

Reply:
This works for me!   Docs describe charts and then release notes describe versions.

Reply:
> I would close this PR and work with @Colin1841 and eng to have that table put and updated on the RN. Sounds good?

Yes, that sounds right to me! Release notes (per patch version) will have the versions there, so users will be able to know which versions they're running if they're on an older patch version.

We do have this page https://docs.d2iq.com/dkp/kommander/2.1/workspaces/applications/platform-applications/ which include version numbers, so we would need to remember to update that page with every patch release; that one would only be tied to the latest patch release of the major/minor version.